src/ruby/lib/symengine.rb

```diff
@@ -0,0 +1,12 @@
+require 'symengine/symengine'
+
+module SymEngine
+  def SymEngine.ascii_art
+    puts "
+ _____           _____         _         
+|   __|_ _ _____|   __|___ ___|_|___ ___ 
+|__   | | |     |   __|   | . | |   | -_|
+|_____|_  |_|_|_|_____|_|_|_  |_|_|_|___|
+      |___|               |___|          "
+  end
+end
```

Comment:
Glad you liked it :smile: ! There are really many options to choose from, and I can see you have tried out almost all of them. The font big and doom look very readable, and the font slant looks pretty cool too! We can shortlist a few and ask others on gitter as to which one looks the best. It's very difficult to decide actually.